### PR TITLE
fix: update wording for the move of the action organisation

### LINKED ISSUE #116: Improve wording around organisation move

### PR DESCRIPTION
**Pull request by gregoryduckworth**
Fixes #116

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,11 +29,17 @@ function logActionRefWarning() {
...
+  if (repoOrg != newOrg) {
     core.warning(
-      `'${repoOrg}' is no longer a valid organisation for this action.` +
-        `Please update it to be under the 'nearform-actions' organisation.`
+      `The '${repoAction}' action, no longer exists under the '${repoOrg}' organisation.\n` +
+        `Please update it to '${newOrg}', you can do this\n` +
+        `by updating your Github Action YAML from:\n\n` +
+        `    uses: '${repoName}@${repoAction}'\n\n` +
+        `to:\n\n` +
+        `    uses: '${newOrg}s@${repoAction}'\n\n`
```
[the remark this answers is not in the document]
Comment (gregoryduckworth) [reply src/index.js]: Fixed this now.

File: src/index.js [hunk 2]
```diff
@@ -27,13 +27,19 @@ function logActionRefWarning() {
  * Displays warning message if the repository is under the nearform organisation
  */
 function logRepoWarning() {
-  const repoName = process.env.GITHUB_ACTION_REPOSITORY
-  const repoOrg = repoName.split('/')[0]
+  const actionRepo = process.env.GITHUB_ACTION_REPOSITORY
+  const repoOrg = actionRepo.split('/')[0]
+  const repoName = actionRepo.split('/')[1]
```
Comment (simoneb) [inline src/index.js:32]:
```suggestion
  const [repoOrg, repoName] = actionRepo.split('/')
```

File: src/index.js [hunk 3]
```diff
@@ -27,13 +27,19 @@ function logActionRefWarning() {
...
+  const newOrg = 'nearform-actions'
 
-  if (repoOrg != 'nearform-actions') {
+  if (repoOrg != newOrg) {
     core.warning(
-      `'${repoOrg}' is no longer a valid organisation for this action.` +
-        `Please update it to be under the 'nearform-actions' organisation.`
+      `The '${repoName}' action, no longer exists under the '${repoOrg}' organisation.\n` +
+        `Please update it to '${newOrg}', you can do this\n` +
+        `by updating your Github Action YAML from:\n\n` +
```
Comment (simoneb) [inline src/index.js:39]: also note we may be able to retrieve the workflow name, but ok not a big deal

```suggestion
        `by updating your GitHub Workflow file from:\n\n` +
```

Comment (gregoryduckworth) [reply src/index.js]: > also note we may be able to retrieve the workflow name, but ok not a big deal

Yeah, I think we can do that with `GITHUB_WORKFLOW`. I'll make a note as a future improvement.

**Review by simoneb — APPROVED** [review]

**Pull request merged**